cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go

```diff
@@ -204,7 +204,7 @@ type NodeRegistrationOptions struct {
 
 	// Taints specifies the taints the Node API object should be registered with. If this field is unset, i.e. nil, in the `kubeadm init` process
 	// it will be defaulted to []v1.Taint{'node-role.kubernetes.io/master=""'}. If you don't want to taint your control-plane node, set this field to an
-	// empty slice, i.e. `taints: {}` in the YAML file. This field is solely used for Node registration.
+	// empty slice, i.e. `taints: []` in the YAML file. This field is solely used for Node registration.
```

Comment:
If I'm not mistaken this fix can apply to the internal API as well...

Reply:
Indeed, let's have it fixed in the internal type too.